src/ResourceManager/ApiManagement/Commands.ApiManagement/Commands/RestoreAzureApiManagement.cs

```diff
@@ -70,18 +70,24 @@ public class RestoreAzureApiManagement : AzureApiManagementCmdletBase
 
         public override void ExecuteCmdlet()
         {
-            var account = StorageContext.GetCloudStorageAccount();
-            var apimanagementResource = Client.RestoreApiManagement(
+            if (CloudStorageAccount.TryParse(StorageContext.ConnectionString, out CloudStorageAccount account))
```

Comment:
What is wrong here @markcowl ? 
` Commands\RestoreAzureApiManagement.cs(73,103): error CS1003: Syntax error, ',' expected`

Reply:
@solankisamir I think this is because we aren't using C# 7 yet, so defining the type of the `out` variable in the method call is not supported yet. If you just define `CloudStorageAccount account = null` above the if-statement and pass in `out account`, I believe it should be fine.